components/MainContentHeader.vue

```diff
@@ -18,6 +18,6 @@ export default {
   <h1>
     {{ title }}
   </h1>
```

Comment:
This is somewhere between specific and hacky. 

Using tailwind to apply specific margins classes that are only needed on the page titles. The element style is applied to avoid the margin reset being overwritten. That extra margin is inherited from the badge and needed on the other instances. 

The specific reset accounts for not having the additional space to the left of the badge when a long title wraps.